src/functions/calculateTargetAllocation.ts

```diff
@@ -0,0 +1,66 @@
+import Allocation, { AssetTypes } from "src/interfaces/allocation"
+
+const EMPTY_TARGETS: Allocation[] = [
+  { type: "celo-native-asset" as const, token: "CELO", percent: 0 },
+  { type: "other-crypto-assets" as const, token: "BTC", percent: 0 },
+  { type: "other-crypto-assets" as const, token: "ETH", percent: 0 },
+  { type: "stable-value" as const, token: "DAI", percent: 0 },
+  { type: "stable-value" as const, token: "USDC", percent: 0 },
+  { type: "natural-capital" as const, token: "cMC02", percent: 0 },
+]
+
+export function calculateTargetAllocation(
+  outstandingStablesUSD: number,
+  totalReserveUSD: number
+): Allocation[] {
+  const targetAllocation: Allocation[] = JSON.parse(JSON.stringify(EMPTY_TARGETS))
+
+  let celoTarget: number
+  let stablesTarget: number
+  let natCapTarget: number
+  let otherCryptoTarget: number
```

Comment:
Should these be initialised to 0?

Reply:
No, this doesn't matter. The check on line 67 will return 0 for values not initialized. This is also a side effect of too much solidity..